lib/rubocop/cop/lint/interpolation_check.rb

```diff
@@ -6,9 +6,8 @@ module Lint
       # Checks for interpolation in a single quoted string.
       #
       # @safety
-      #   This cop is generally safe, but is marked as unsafe because
-      #   it is possible to actually intentionally have text inside
-      #   `#{...}` in a single quoted string.
+      #   This cop's autocorrection is unsafe because although it always replaces single quotes as
```

Comment:
I think we need some example here that illustrates what's unsafe, as they way it's currently worded is not very clear IMO.